examples/camera-app/linux/include/webrtc-transport.h

```diff
@@ -93,7 +93,8 @@ class WebrtcTransport : public Transport
     // Stops WebRTC peer connection and cleanup
     void Stop();
 
-    void AddTracks(const std::string & videoMid = "video", const std::string & audioMid = "audio");
+    void AddVideoTrack(const std::string & videoMid = "video", int payloadType = 96);
+    void AddAudioTrack(const std::string & audioMid = "audio", int payloadType = 111);
```

Comment:
what default value 96 and 111 means? should we use constant instead

Reply:
Thank you for your review. It is H264 default value and Opus codec default value. we added comments on the line.